fake_joint_driver/src/fake_joint_driver_node.cpp

```diff
@@ -6,36 +6,62 @@
  *
  * Device driver node to fake loopback joints.
  */
-#include "ros/ros.h"
-#include "controller_manager/controller_manager.h"
-#include "fake_joint_driver/fake_joint_driver.h"
+#include <controller_manager/controller_manager.hpp>
+#include <fake_joint_driver/fake_joint_driver.h>
+#include <rclcpp/rclcpp.hpp>
+
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("fake_joint_driver");
+
+void spin(std::shared_ptr<rclcpp::executors::MultiThreadedExecutor> exe)
+{
+  exe->spin();
+}
 
 /**
  * @brief Main function
  */
-int main(int argc, char **argv)
-{
+int main(int argc, char **argv) {
   // Init ROS node
-  ros::init(argc, argv, "fake_joint_driver");
-  ros::NodeHandle nh;
-
+  rclcpp::init(argc, argv);
+  auto executor = std::make_shared<rclcpp::executors::MultiThreadedExecutor>();
```

Comment:
Why use a shared pointer?

Reply:
It's because of the controller manager takes a shared pointer

Reply:
Sorry I forgot to answer your last comment about this